DFReader.py

```diff
@@ -64,6 +81,9 @@ def __init__(self, type, name, flen, format, columns, oldfmt=None):
         self.unit_ids = None
         self.mult_ids = None
 
+        if len(self.columns) != len(format):
+            raise(FormatColumnCountMismatch(format, columns))
```

Comment:
This makes me question the two lines below. Can we have a FMT message without format?